client/dplug/client/eventclient.d

```diff
@@ -0,0 +1,75 @@
+/*
+Cockos WDL License
+
+Copyright (C) 2005 - 2015 Cockos Incorporated
+Copyright (C) 2015 - 2017 Auburn Sounds
+
+Portions copyright other contributors, see each source file for more information
+
+This software is provided 'as-is', without any express or implied warranty.  In no event will the authors be held liable for any damages arising from the use of this software.
+
+Permission is granted to anyone to use this software for any purpose, including commercial applications, and to alter it and redistribute it freely, subject to the following restrictions:
+
+1. The origin of this software must not be misrepresented; you must not claim that you wrote the original software. If you use this software in a product, an acknowledgment in the product documentation would be appreciated but is not required.
+1. Altered source versions must be plainly marked as such, and must not be misrepresented as being the original software.
+1. This notice may not be removed or altered from any source distribution.
+*/
```

Comment:
Guess this is fine, since it extends `dplug.client.client` that falls under the WDL license.